vignettes/prudence.Rmd

```diff
@@ -198,7 +233,7 @@ flights_frugal[[1]]
 ```
 
 
-### Enforcing DuckDB operation
+### Side effect: Enforcing DuckDB operation
```

Comment:
It is an important section but also does not go with the flow of the rest of the vignette.

Reply:
Good point, moving to "fallback".

Reply:
In the PR where I edit the fallback vignette, I added a section about this but that refers to this vignette.